File: templates/problems/09_heap/06_design_twitter/go/design_twitter.go
package design_twitter

type Twitter struct {
}

func ConstructorTwitter() Twitter {
	return Twitter{}
}

func (t *Twitter) PostTweet(userId int, tweetId int) {
}

func (t *Twitter) GetNewsFeed(userId int) []int {
	return nil
}

func (t *Twitter) Follow(followerId int, followeeId int) {
}

func (t *Twitter) Unfollow(followerId int, followeeId int) {
}

File: templates/problems/09_heap/06_design_twitter/typescript/design_twitter.ts
export class Twitter {
  constructor() {}

  postTweet(userId: number, tweetId: number): void {}

  getNewsFeed(userId: number): number[] {
    return [];
  }

  follow(followerId: number, followeeId: number): void {}

  unfollow(followerId: number, followeeId: number): void {}
}
